src/controllers/session/login/index.html

```diff
@@ -9,7 +9,12 @@ <h1 class="sectionTitle">${HeaderPleaseSignIn}</h1>
             </div>
 
             <div class="inputContainer">
-                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password" />
+                <input is="emby-input" id="txtManualPassword" type="password" label="${LabelPassword}" autocomplete="current-password">
+                <button class="passwordToggle" type="button">
+                    <span class="material-icons">
+                        visibility
```

Comment:
Iirc, ligatures don't work on some webOS. That's why we use icons in the `class` attribute.
https://github.com/jellyfin/jellyfin-web/pull/814